packages/next/src/build/utils.ts

```diff
@@ -404,12 +404,22 @@ export async function printTreeView(
     gzipSize?: boolean
   }
 ) {
-  const getPrettySize = (_size: number): string => {
-    const size = prettyBytes(_size)
-    return white(bold(size))
+  const getPrettySize = (
+    _size: number,
+    { strong }: { strong?: boolean } = {}
+  ): string => {
+    const size = process.env.__NEXT_PRIVATE_DETERMINISTIC_BUILD_OUTPUT
+      ? '42 kB' // Use a fake fixed size to enforce deterministic build output.
```

Comment:
I'd choose a more explicit placeholder to make it clear these aren't actual numbers e.g. `N kB`